src/Apps/InfiniteDiscovery/InfiniteDiscoveryApp.tsx

```diff
@@ -0,0 +1,310 @@
+    console.log("curatedArtworks", curatedArtworks)
+    console.log("likedArtworks", likedArtworks)
+    console.log("dismissedArtworks", dismissedArtworks)
+
+    console.log(data)
+    const artworksResponse = data?.hits?.hits.map(hit => hit._id)
+
+    const aArtworks = artworksResponse.slice(0, 8)
+    const cArtworks = sampleSize(curatedArtworks, 2)
+
+    console.log("aArtworks", aArtworks)
+    console.log("cArtworks", cArtworks)
+
+    setArtworks(shuffle([...aArtworks, ...cArtworks]))
+  }
+
+  useEffect(() => {
+    const req = async () => {
+      const response = await request(getMetaphysicsEndpoint(), {
+        body: JSON.stringify({ query: CURATORS_PICKS_QUERY }),
+        mode: "no-cors",
+      })
+
+      const artworks = uniq(response?.data?.marketingCollection?.artworkIds)
+      setArtworks(sampleSize(artworks, 10))
+      setCuratedArtworks(artworks)
+    }
+    req()
+  }, [])
+
+  if (artworks.length === 0) {
+    return <h1>Nothing to show</h1>
+  }
+
+  return (
+    <Flex flexDirection="column">
+      <Flex flexDirection="column">
+        <div>
+          <br />
+          <Text>
+            Welcome to the Infinite Discovery Notebook! Like or dismiss artworks
+            to receive personalized recommendations tailored to your
+            preferences.
+          </Text>
+          <br />
+          <div>
+            <Button
+              onClick={submitSearch}
+              disabled={
+                likedArtworks.length === 0 && dismissedArtworks.length === 0
+              }
+            >
+              Get Recommendations
+            </Button>
+            <br />
+            <br />
+            <br />
+          </div>
+        </div>
+      </Flex>
+      <Flex justifyContent={"space-between"} flexWrap={"wrap"}>
+        {artworks.map((artwork: any) => {
+          return (
+            <Artwork
+              key={artwork}
+              internalID={artwork}
+              onLike={onLike}
+              onDismiss={onDismiss}
+              isCurated={isCurated}
+            />
+          )
+        })}
+      </Flex>
+      <Separator />
+      <Flex flexDirection={"column"}>
+        <Text fontSize={15}>
+          <b>Liked artworks:</b> {likedArtworks.join(", ")}
+        </Text>
+        <Text fontSize={15}>
+          <b>Dismissed artworks:</b> {dismissedArtworks.join(", ")}
+        </Text>
+      </Flex>
+    </Flex>
+  )
+}
+
+const Artwork = ({ internalID, onLike, onDismiss, isCurated }) => {
+  const [artwork, setArtwork] = React.useState({}) as any
+
+  useEffect(() => {
+    const req = async () => {
+      const response = await request(getMetaphysicsEndpoint(), {
+        body: JSON.stringify({ query: ARTWORK_QUERY(internalID) }),
+        mode: "no-cors",
+      })
+      if (!response?.data?.artwork) {
+        return setArtwork({
+          title: "Artwork not available",
+        })
+      }
+      setArtwork(response?.data?.artwork)
+    }
+    req()
+  }, []) // eslint-disable-line
```

Comment:
Assuming this is to silence the missing dependency error . If so, any reason we don't want to declare the dependency?